website/docs/cloud-docs/api-docs/audit-trails.mdx

```diff
@@ -108,23 +108,334 @@ $ curl \
+| `policy_check`             | `override`                                                   | Events related to policy checks.                           |
+| `policy_config`            | `create`                                                     | Events related to policy configurations.                  |
+| `policy_set`               | `destroy`                                                    | Events related to policy sets.                             |
+| `policy_version`           | `create`                                                     | Events related to policy versions.                         |
+| `var` (policy set parameters)           | `destroy`                                                     | Events related to destroying policy set parameters                         |
+| `project`                  | `create`, `update`, `destroy`                               | Events related to projects.                                |
+| `registry_module`          | `destroy`, `update`                                          | Events related to registry modules.                        |
+| `registry_provider`        | `create`, `destroy`                                          | Events related to registry providers.                      |
+| `registry_provider_platform` | `create`, `destroy`                                          | Events related to registry provider platforms.             |
+| `registry_provider_version` | `create`, `destroy`                                          | Events related to registry provider versions.             |
+| `run`                      | `apply`, `cancel`, `force_cancel`, `discard`, `force_execute`, `create` | Events related to runs.                                    |
+| `run_trigger`              | `create`, `destroy`                                          | Events related to run triggers.                            |
+| `saml_configuration`       | `create`, `update`, `destroy`, `enable`, `disable`          | Events related to SAML configurations.                    |
+| `ssh_key`                  | `index`, `show`, `create`, `update`, `destroy`              | Events related to SSH keys.                                |
+| `stack`                     | `create`                                                    | Events related to creating stacks.                         |
+| `state_version`            | `index`, `show`, `create`, `soft_delete_backing_data`, `restore_backing_data`, `permanently_delete_backing_data` | Events related to state versions.                         |
+| `task_stage`                  | `override`              | Events related to an overriden task stage.                                |
+| `user`                  | `index`              | Events related to indexing users.                                |
+| `var`                  | `index`, `show`, `create`, `update`, `destroy`              | Events related to variables.                             |
```

Comment:
@carolinaborim There were two "vars" on the sheet, one for policy set parameters (which I indicated above), and the second one didn't have any additional info. I assume this was referring to variables? Maybe environment variables?

Reply:
Yes. environment variables that we want to set for the infra.

Reply:
Sweet, thank you! I'll add that!